package.json

```diff
@@ -39,7 +39,7 @@
     "@types/vuelidate": "^0.7.11",
     "@typescript-eslint/eslint-plugin": "^2.23.0",
     "@typescript-eslint/parser": "^2.23.0",
-    "@vue/cli-plugin-babel": "4.2.3",
+    "@vue/cli-plugin-babel": "4.4.1",
```

Comment:
Please call out this change in the PR notes.

Also please update this project's version (line 3 above).

Reply:
I noted it in change description

kk, will do 👍 .

Reply:
Done, thanks!